docs/running.md

```diff
@@ -32,11 +34,21 @@ docker run -v $STEWARD_DIR:/opt/scala-steward -it scala-steward:0.1.0-SNAPSHOT \
   --env-var FOO=BAR
 ```
 
-If you run Scala Steward for your own private projects, you can pass additional environment variables from the command line using the `--env-var` flag as shown in the examples above. You can use this to pass any credentials required by your projects to resolve any private dependencies, e.g.:
+The [`git-ask-pass` option](https://git-scm.com/docs/gitcredentials) must specify an executable file (script) that returns (on the stdout),
+
+- either the plain text password corresponding to the configured `${LOGIN}`,
+- or (recommanded) an authentication token corresponding to `${LOGIN}` (with appropriate permissions to watch the repositories; e.g. [Create a personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) for GitHub).
+
+### Private repositories
+
+If you run Scala Steward for your own private projects, the option `--do-not-fork` can be required, not to fork.
+Instead it will create pull requests directly on the private repository (as soon as the `${LOGIN}` can).
+
+It can also be useful to pass additional environment variables from the command line using the `--env-var` flag as shown in the examples above. You can use this to pass any credentials required by your projects to resolve any private dependencies, e.g.:
 
 ```bash
 --env-var BINTRAY_USER=username \
 --env-var BINTRAY_PASS=password
 ```
 
-These variables will be accessible (in sbt) to all of the projects that Scala Steward checks dependencies for.
+These variables will be accessible (in SBT) to all of the projects that Scala Steward checks dependencies for.
```

Comment:
[It should just be sbt.](https://www.scala-sbt.org/1.x/docs/Faq.html#What+does+the+name+%E2%80%9Csbt%E2%80%9D+stand+for%2C+and+why+shouldn%E2%80%99t+it+be+written+%E2%80%9CSBT%E2%80%9D%3F) :-)
```suggestion
These variables will be accessible (in sbt) to all of the projects that Scala Steward checks dependencies for.
```

Reply:
Weird naming ... updated